pwiz/data/msdata/Diff.hpp

```diff
@@ -228,14 +228,17 @@ struct PWIZ_API_DECL DiffConfig : public pwiz::data::BaseDiffConfig
 
     bool ignoreDataProcessing;
 
+    bool ignoreSpectrumTitle;
+
```

Comment:
Why is this necessary? I would expect the new CCS-in-title code to round trip the CCS so the title will be preserved and not different. What diffs were you getting without this?

Reply:
Sorry I dropped the ball here. What did you mean "the reference does not contain the text"?